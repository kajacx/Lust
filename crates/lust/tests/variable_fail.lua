-- @type nil
local flag = true

-- @type boolean
local value = 8

-- @type number
local text = "Hello world!"

-- @type string
local empty = nil

-- @type boolean
local textAgain = text
