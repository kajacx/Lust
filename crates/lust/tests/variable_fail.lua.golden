tests/variable_fail.lua:2:1: Cannot assign value true of type true into the variable flag of type nil.
tests/variable_fail.lua:5:1: Cannot assign value 8 of type number into the variable value of type (boolean).
tests/variable_fail.lua:8:1: Cannot assign value "Hello world!" of type string into the variable text of type number.
tests/variable_fail.lua:11:1: Cannot assign value nil of type nil into the variable empty of type string.
tests/variable_fail.lua:14:1: Cannot assign value text of type number into the variable textAgain of type (boolean).
